lib/controler.rb

```diff
@@ -7,7 +7,12 @@ def initialize(user_interface, actions)
   end
 
   def start
-    actions[user_interface.menu_choice - 1].run
+    loop do
+      user_choice = user_interface.menu_choice
+
+      actions[user_choice - 1].run
+      break if user_choice == actions.length
```

Comment:
Require the user interface at the top and then use `ClassName::CONSTANT_NAME` and it should work (if you require the user interface in the controller you can delete the user interface require in the test)